src/fmu/ensemble/realizationcombination.py

```diff
@@ -117,16 +120,36 @@ def get_df(self, localpath):
             return result.reset_index()
         return result.dropna().to_dict()
 
-    def to_virtual(self):
+    def to_virtual(self, keyfilter=None):
         """Evaluate the current linear combination and return as
         a virtualrealizatione.
+
+        Args:
+            keyfilter (list or str): If supplied, only keys matching wildcards
+                in this argument will be included. Use this for speed reasons
+                when only some data is needed. Default is to include everything.
+                If you supply "unsmry", it will match everthing key that
+               includes this string by prepending and appending '*' to your pattern
+
+        Returns:
+            VirtualRealization
         """
         # pylint: disable=import-outside-toplevel
         from .virtualrealization import VirtualRealization
 
+        if keyfilter is None:
+            keyfilter = "*"
+        if isinstance(keyfilter, six.string_types):
+            keyfilter = [keyfilter]
+        if not isinstance(keyfilter, list):
+            raise TypeError("keyfilter in to_filter() must be list or string")
```

Comment:
Same as in ensemblecombination.py:
```suggestion
            raise TypeError("keyfilter in to_virtual() must be list or string")
```